### PR TITLE
Issue 1569: Integration Tests fail in strongbox-web-core due to messa…

### LINKED ISSUE #1569: Integration Tests fail in strongbox-web-core due to message mismatch with non english locale
# Bug Description

Integration tests in strongbox-web-core fail when run on a system with non english locale.

# Steps To Reproduce

1. Clone repo
2. Set system locale to something other than english (tested on german)
3. Build repo with `mvn clean install -Dintegration.tests`

# Expected Behavior
Tests should not be affected by locale of the building system

# Environment
* Strongbox Version: 1.0-SNAPSHOT
* OS: Windows 10 x64 - German
* Maven version (please provide the full output of `mvn --version`):

`Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T21:00:29+02:00)
Maven home: C:\Program Files\JetBrains\IntelliJ IDEA 2019.2\plugins\maven\lib\maven3
Java version: 1.8.0_202, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jre1.8.0_202
Default locale: de_AT, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"`

# Proposed Solution 
Implement fix similar to [Issue 1466/Pull 1467](https://github.com/strongbox/strongbox/pull/1467)

# Known Workarounds
Change system locale to english

# Useful Links 

* [Similar Issue](https://github.com/strongbox/strongbox/issues/1466)

### PR DESCRIPTION
**Pull request by bczyzowski**
# Pull Request Description

This pull request closes #1569 

# Acceptance Test

* [x] Building the code with `mvn clean install -Dintegration.tests` still works.
* [x] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [x] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [x] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

### CODE REVIEW

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution!

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Kind regards,

Martin

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java [hunk 1]
```diff
@@ -362,4 +365,19 @@ void testRemoteRepositoryFormInvalidNegativeHttpConnectionPool()
 
         validateAndAssert(repositoryForm, 1, "A httpConnectionPool must be positive or zero.");
     }
+
+    private String getPatternLocalisedMessage(String expectedPattern)
+    {
+        String message;
+        try {
```
Comment (carlspring) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Cosmetic: Braces on new lines please. :)

For more details, please check our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html). Thanks! :)

**Comment by carlspring** [comment]
@mkisser-tgm,

As you were the one who reported it, would you mind testing this fix? Thanks! :)

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java (hunk 1 above)
Comment (bczyzowski) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Formatted

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java [hunk 2]
```diff
@@ -362,4 +365,22 @@ void testRemoteRepositoryFormInvalidNegativeHttpConnectionPool()
...
     }
+
+    private String getPatternLocalisedMessage(String expectedPattern)
+    {
+        String message;
+        try
+        {
+            final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
+                    .getResourceBundle(Locale.getDefault())
+                    .getString("javax.validation.constraints.Pattern.message");
```
Comment (ptirador) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Please, format this code according to [our coding convention](https://strongbox.github.io/developer-guide/coding-convention.html). Thanks! :)

Comment (bczyzowski) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Formatted

Comment (ptirador) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Sorry, but that I don't think that this is well formated by using our code style formatter. It should be like this:
https://gist.github.com/ptirador/515a10e9bf752013e4bb8ff61121f9a1

Thanks.

Comment (bczyzowski) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Commit bd9592cbfd364ba4480b464e73f73f660d0163af fixes this issue.

Comment (ptirador) [reply strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java]: Please take a look at what I said. It's not exactly well formated.

**Comment by carlspring** [comment]
@bczyzowski ,

Thank you for your contribution! I will wait for @mkisser-tgm to try it out, before merging it.

**Comment by carlspring** [comment]
@mkisser-tgm ,

(Ping): Could you please test this?

**Comment by carlspring** [comment]
Looks good to me. I'm merging it as it is. If anyone has locale issues, they can report them and we can re-visit this, if there are any problems.

**Pull request merged**